tests/e2e/cli/instance_segmentation/test_tiling_instseg.py

```diff
@@ -202,18 +201,6 @@ def test_nncf_optimize(self, template, tmp_dir_path):
 
     @e2e_pytest_component
     @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
-    @pytest.mark.skip("Issue#2060: Tiling nncf shows errors")
-    @pytest.mark.parametrize("template", templates, ids=templates_ids)
-    def test_nncf_export(self, template, tmp_dir_path):
```

Comment:
Is removing `test_nncf_export` your intention?

Reply:
I recover that